src/ocrd_anybaseocr/cli/ocrd_anybaseocr_layout_analysis.py

```diff
@@ -96,7 +97,8 @@ def process_page_file(self, *input_files : Optional[OcrdFileType]) -> None:
         self.page_labels[page_id] = self._predict(img_array)
 
     def shutdown(self) -> None:
-        self.reset()
+        del self.model
+        del self.label_mapping
```

Comment:
These might never have been initialized, or the GC might already have struck:
```suggestion
        if hasattr(self, 'model'):
            del self.model
        if hasattr(self, 'label_mapping'):
            del self.label_mapping
```